submarine-cloud-v2/main.go

```diff
@@ -19,17 +19,18 @@ package main
 
 import (
 	"flag"
-	kubeinformers "k8s.io/client-go/informers"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
-	"k8s.io/klog/v2"
```

Comment:
Why do we change these lines? My code formatter `gofmt` does not change these lines.